### PR TITLE
[IMP] mrp: documentation major update

### PR DESCRIPTION
**Pull request by amoyaux**
- Updated Pictures with latest version, deleted old ones
- Added Quality Control section
- Deleted Kit Routings section
- Adapted Terminology, specifically work orders vs routings

### CODE REVIEW

**Comment by AntoineVDV** [comment]
@amoyaux Given that the diff is quite significant I'll let @sdr-odoo post a content review and ping me when it's all good ;)

**Comment by amoyaux** [comment]
Very good

File: content/applications/inventory_and_mrp/manufacturing/management/subcontracting.rst
Comment (sdr-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/subcontracting.rst:146]: I would maybe say "move line hamburger menus" because I was very confused for a few minutes :)

File: content/applications/inventory_and_mrp/manufacturing/management/subcontracting.rst
Comment (sdr-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/subcontracting.rst:153]: change "wether" to "whether"

File: content/applications/inventory_and_mrp/manufacturing/management/subcontracting.rst
Comment (sdr-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/subcontracting.rst:154]: fix type, change to "described"

File: content/applications/inventory_and_mrp/manufacturing/management/subcontracting.rst
Comment (sdr-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/subcontracting.rst:173]: I think it's Purchase > Configuration > Settings > Dropshipping

File: content/applications/inventory_and_mrp/manufacturing/management/subcontracting.rst
Comment (sdr-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/subcontracting.rst:187]: Same thing, I would add in Settings after Configuration

File: content/applications/inventory_and_mrp/manufacturing/management/subcontracting.rst
Comment (sdr-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/subcontracting.rst:200]: I don't think you need "however"

File: content/applications/inventory_and_mrp/manufacturing/management/subcontracting.rst
Comment (sdr-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/subcontracting.rst:202]: delete the extra "at" in this sentence

File: content/applications/inventory_and_mrp/manufacturing/management/subcontracting.rst
Comment (sdr-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/subcontracting.rst:211]: change "adress" typo to "address"

File: content/applications/inventory_and_mrp/manufacturing/management/use_mps.rst
Comment (sdr-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/use_mps.rst:15]: change "setting" to "settings"

**Review by sdr-odoo — CHANGES_REQUESTED** [review]
@amoyaux Finished the content review and double-checked all the flows :). My feedback is mostly about cleaning up some sentences/making a few things more clear but there are also some typos that should be corrected.  Let me know if you have questions!

**Comment by AntoineVDV** [comment]
@sdr-odoo Pro tip for the reviews: you can hit the "suggestion" button or use `ctrl+g` when typing a comment to suggest a content change for the line(s) you're commenting on.

File: content/applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst
Comment (mathieu-mgm) [reply content/applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst:9]: ok

**Comment by sdr-odoo** [comment]
@AntoineVDV Got it, thank you. I will do that in the future!

File: content/applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst
Comment (mathieu-mgm) [reply content/applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst:13]: Modified it a little bit further as it sounded repetitive with the first section.

File: content/applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst
Comment (mathieu-mgm) [reply content/applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst:16]: removed the 'productive' concept, as this is rather related to an operation versus the initial planning concept which we are targeting here.

File: content/applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst
Comment (mathieu-mgm) [reply content/applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst:23]: ok

File: content/applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst
Comment (mathieu-mgm) [reply content/applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst:39]: ok

File: content/applications/inventory_and_mrp/manufacturing/management/subcontracting.rst
Comment (mathieu-mgm) [reply content/applications/inventory_and_mrp/manufacturing/management/subcontracting.rst:187]: good point

File: content/applications/inventory_and_mrp/manufacturing/management/subcontracting.rst
Comment (mathieu-mgm) [reply content/applications/inventory_and_mrp/manufacturing/management/subcontracting.rst:200]: indeed

File: content/applications/inventory_and_mrp/manufacturing/management/subcontracting.rst
Comment (mathieu-mgm) [reply content/applications/inventory_and_mrp/manufacturing/management/subcontracting.rst:202]: yes

File: content/applications/inventory_and_mrp/manufacturing/management/use_mps.rst
Comment (mathieu-mgm) [reply content/applications/inventory_and_mrp/manufacturing/management/use_mps.rst:15]: done

**Comment by mathieu-mgm** [comment]
@sdr-odoo applied all changes, looks good to me

**Review by sdr-odoo — APPROVED** [review]

**Comment by sdr-odoo** [comment]
@AntoineVDV Looks good now! Mathieu made all the requested changes and I don't have any more edits :).

**Comment by AntoineVDV** [comment]
> @AntoineVDV Looks good now! Mathieu made all the requested changes and I don't have any more edits :).

Good, then please request a review from `@odoo/doc-review` :)
This way, anyone from this team can review and it's easier for us to manage pending reviews.

File: content/applications/inventory_and_mrp/manufacturing/management.rst
Comment (AntoineVDV) [inline content/applications/inventory_and_mrp/manufacturing/management.rst:16]: Is the content of this deleted page moved elsewhere? If so, you need to create a redirection rule. See https://github.com/odoo/documentation/blob/15.0/redirects.txt

File: content/applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst
Comment (AntoineVDV) [inline content/applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst:12]: Take care to [limit the lines' length to 100 characters](https://www.odoo.com/documentation/master/contributing/documentation/rst_guidelines.html#start-a-new-line-before-the-100th-character).
```suggestion
which work center is busy and which equivalent ones are free, and schedules orders and maintenance
accordingly.
```

File: content/applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst
Comment (AntoineVDV) [inline content/applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst:20]: Use the `menuselection` directive. See H2 of https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html#document-structure

File: content/applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst
Comment (AntoineVDV) [inline content/applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst:24]: Make sure that all new (or reworked) images have:
- a consistent file name
- the right location: place the file inside a folder of the same name as the doc file referring to it (pick the folder that makes the most sense if two pages are referring to the same image)
- an ALT tag

See https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html#images

Comment (AntoineVDV) [reply content/applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst:24]: Also, please run `pngquant` on all these images to reduce their weight by a good 2/3rd.

File: content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst
Comment (AntoineVDV) [inline content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst:19]: Use the `doc` directive for relative paths: https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#the-doc-directive

**Review by AntoineVDV — CHANGES_REQUESTED** [review]

**Comment by AntoineVDV** [comment]
Hi, @sdr-odoo I see that you requested a review from `odoo/manufacturing-doc-review` (yourself). The appropriate group to request the final (technical) review from after the content has been approved by marketing is `odoo/doc-review`.

Also, please take a look at the review comment above. They will be useful for your next reviews :)

**Comment by amoyaux** [comment]
@AntoineVDV

File: content/applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst
Comment (AntoineVDV) [inline content/applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst:21]: `menuselection` needs selection items to be separated by `-->` and not `>`. Otherwise, you are creating a selection of a single item.

File: content/applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst
Comment (AntoineVDV) [inline content/applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst:38]:
```suggestion
Next, go to :menuselection:`Products --> Bills of Materials`, choose your product, 
```

File: content/applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst
Comment (AntoineVDV) [inline content/applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst:51]:
```suggestion
Go to :menuselection:`Operations --> Manufacturing Orders`, and hit **Create**. There, add
```

File: content/applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst
Comment (AntoineVDV) [inline content/applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst:29]:
```suggestion
Now, go to :menuselection:`Configuration --> Work Centers` and create
```

File: content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst
Comment (AntoineVDV) [inline content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst:22]:
```suggestion
To create a BoM from the Manufacturing module, go to :menuselection:`Products --> Bills of Materials`. 
```

File: content/applications/inventory_and_mrp/manufacturing/management/product_variants.rst
Comment (AntoineVDV) [inline content/applications/inventory_and_mrp/manufacturing/management/product_variants.rst:7]: Spaces between the `menuselection` directive and the selection break the directive.
```suggestion
the same product. To activate variants, simply go to :menuselection:`Configuration --> Settings
--> Variants`.
```

File: content/applications/inventory_and_mrp/manufacturing/management/quality_control.rst
Comment (AntoineVDV) [inline content/applications/inventory_and_mrp/manufacturing/management/quality_control.rst:11]: menuselection

File: content/applications/inventory_and_mrp/manufacturing/management/subcontracting.rst
Comment (AntoineVDV) [inline content/applications/inventory_and_mrp/manufacturing/management/subcontracting.rst:215]:
```suggestion
```

File: content/applications/inventory_and_mrp/manufacturing/management/use_mps.rst
Comment (AntoineVDV) [inline content/applications/inventory_and_mrp/manufacturing/management/use_mps.rst:76]:
```suggestion
-  **Orange**: replenishment order has already been generated, and its quantity was too low considering current data.
```

**Review by AntoineVDV — CHANGES_REQUESTED** [review]

**Comment by amoyaux** [comment]
Done

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**
